src/App.js

```diff
@@ -0,0 +1,21 @@
+import { Provider } from 'react-redux';
+import { Route, Routes } from 'react-router-dom';
+import store from './redux/configureStore';
+import Home from './pages/Home';
+import Details from './components/Details';
+import './styling/App.css';
+
+function App() {
+  return (
+    <Provider store={store}>
+      <div className="App">
+        <Routes>
+          <Route path="/" element={<Home />} />
+          <Route path="/details" element={<Details />} />
+        </Routes>
```

Comment:
- [ ] It will be good for each country's detail page to have its unique route within the SPA. e.g. If I click on the Tunisia country card the URL should be `https://metrics-covid19.netlify.app/details/Tunisia`, I suggest using the React Router hook([useParams](https://reactrouter.com/docs/en/v6/api#useparams)).